config.test.yaml

```diff
@@ -159,6 +163,83 @@ templates:
                   options:
                     host: http://appservice.wmflabs.org
 
+      /{module:summary}:
```

Comment:
I think this came back in the rebase before my push last night. I swear I had deleted it already. In any case, it was dead code & is now removed.